### DISCUSSION TITLE
How to Create Column with Setting value

### DISCUSSION THREAD

**Discussion by KKimj** [AntlerVC/firetable#870] [Support / Q&A] [answered]
Hello, Rowy team!
Thanks your OSS Project!
![image](https://user-images.githubusercontent.com/40026920/196172203-5ec31187-2cc1-4e2f-958e-198f7a7cf2c1.png)

I want to add a new column to the data table.
When it is a boolean value, I want to explicitly store it in the Firestore as a `true` value.

Something like below!
![image](https://user-images.githubusercontent.com/40026920/196172452-05016736-59e3-4bbd-80f7-3ae2d759c219.png)
I wonder if the only way is to set it up manually one by one!

![image](https://user-images.githubusercontent.com/40026920/196172646-9ebb228e-4675-4c4e-9bd5-90a5239dc527.png)

I've already used column config, but I couldn't do the operation I wanted.
I couldn't resolve it by myself.
I'd appreciate your help.

Thanks!
Best,

**Comment by BeeBombshell** [comment, marked as answer]
Hi @KKimj, glad you are enjoying the project! 😄 
I went through the issue mentioned above. Just wanted to let you know that when you set a toggle to a default `true` or `false` state, the rows entered after this initialization will have the toggle set accordingly.
